site/en/workflows.md

```diff
@@ -1217,19 +1244,146 @@ copies them the a new Output fileGrp, re-generating the PAGE XML from the curren
+
+#### Available processors
+
+<table class="processor-table">
+  <thead>
+    <tr>
+      <th>Processor</th>
+      <th>Parameter</th>
+      <th>Remarks</th>
+    <th>Call</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr data-processor="ocrd-tesserocr-fontshape">
+      <td>ocrd-tesserocr-fontshape</td>
+      <td><code>-P model osd -P padding 2</code></td>
+      <td></td>
+      <td><code>ocrd-tesserocr-fontshape -I SEG-WORD -O SEG-WORD-FONT</code></td>
+    </tr>
+    <tr data-processor="ocrd-typegroups-classifier">
+      <td>ocrd-typegroups-classifier</td>
+      <td><code>-P network /path/to/densenet121.tgc</code></td>
+      <td>Download [`densenet121.tgc` from GitHub](https://github.com/seuretm/ocrd_typegroups_classifier/raw/master/ocrd_typegroups_classifier/models/densenet121.tgc)</td>
+      <td><code>ocrd-typegroups-classifier -I OCR-D-IMG -O OCR-D-IMG-FONTS</code></td>
+    </tr>
+  </tbody>
+
+<!-- END-EVAL -->
+
 # Recommendations
 
 <!-- BEGIN-INCLUDE ./repo/ocrd-website.wiki/Workflow-Guide-recommendations.md -->
-All processors, with the exception of those for post-correction, were tested on
-selected pages of some prints from the 17th and 18th century.
+In order to facilitate the usage of OCR-D and the configuration of workflows, we provide two workflows
+which can be used as a start for your OCR-D-tests. They were determined by testing the processors listed
+above on selected pages of some prints from the 17th and 18th century.
 
 The results vary quite a lot from page to page. In most cases, segmentation is a problem.
 
-These recommendations may also work well for other prints of those centuries.
-
 Note that for our test pages, not all steps described above werde needed to obtain the best results.
 Depending on your particular images, you might want to include those processors again for better results.
 
+We are currently working on regression tests with the help of which we will be able to provide more profound
+workflows soon, which will replace those interm solutions. 
+
+## Minimal workflow
+
+Since `ocrd-tesserocr-recognize` can do all line segmentation, region
+segmentation, table detection and even binarization (Otsu), just like the
+upstream `tesseract` command line tool, it's a good single-step workflow to get
+a baseline result to compare to granular workflows.
+
+**Note:** Be aware that you will most likely obtain significantly better
+results by configuring a more granular workflow like e.g. the
+[workflows](#best-results-for-selected-pages)
+[below](#good-results-for-slower-processors).
+
+<table class="processor-table">
+  <thead>
+    <tr>
+      <th>Step</th>
+      <th>Processor</th>
+      <th>Parameter</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>1</td>
+      <td>ocrd-tesserocr-recognize</td>
+      <td>-P segmentation_level line -P textequiv_level word -P find_tables true -P overwrite_segments true -P model GT4HistOCR_50000000.997_191951</td>
```

Comment:
Note: `segmentation_level` is the **entry point** to segmentation; if you had no regions already, then `line` is too low and nothing will happen. I would disable and hide `overwrite_segments` here (we don't expect any existing segments).

```suggestion
      <td>-P segmentation_level region -P textequiv_level word -P find_tables true -P model GT4HistOCR_50000000.997_191951</td>
```